packages/ChangesReloaded-Core.package/CRBrowser.class/instance/ignoreAllOfHistoryListExceptSelected.st

```diff
@@ -0,0 +1,9 @@
+menu-actions
+ignoreAllOfHistoryListExceptSelected
+
+	| selectedHistoryListItem |
+	selectedHistoryListItem := self selectedHistoryListItem.
```

Comment:
Why the temp var? In my opinion, `each = self selectedHistoryListItem` would be okay, too.

Reply:
Historisch gewachsen